mp4parse_capi/src/lib.rs

```diff
@@ -741,13 +741,12 @@ fn create_sample_table(track: &Track) -> Option<Vec<mp4parser_indice>> {
                 }
             true
         });
-    if !ret {
+    if sample_table.is_empty() {
```

Comment:
OK, I change it to a for loop. I think it's probably better to use ```.map()``` to where nothing could be wrong.